server/sonar-server/src/main/java/org/sonar/server/qualityprofile/QProfileReset.java

```diff
@@ -112,7 +116,6 @@ BulkChangeResult reset(QProfileName profileName, Collection<RuleActivation> acti
     try {
       QualityProfileDto profile = factory.getOrCreate(dbSession, profileName);
       BulkChangeResult result = doReset(dbSession, profile, activations);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "result". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1488)